config_example.json

```diff
@@ -1,5 +1,6 @@
 {
  "name": "Skynet",
+ "dataDirectory":"data",
```

Comment:
If someone does: `.\gocryptotrader.exe -config="config_example.json"`
It will populate a local data folder which I see your point on keeping it relative with the configuration but will leave this open for more discussion, I think for now it's probably better to just change to `"dataDirectory":"",` as this will just use default folder settings. I personally like to have multiple configurations and a centralised data directory as default if not specified.

Reply:
Done.